File: Reports/Daniil_Pozdnyakov/lab4/src/task3/Person.cs
namespace lab4._3._5
{
    public class Person
    {
        private int Age = 0;
        private string Name;

        public Person(int age, string name)
        {
            Age = age;
            Name = name;
        }

        public void f_set_age(int age)
        {
            Age = age;
        }

        public int f_get_age()
        {
            return Age;
        }

        public void f_set_name(string name)
        {
            Name = name;
        }

        public string f_get_name()
        {
            return Name;
        }
    }
}

File: reports/Ignatyuk/Lab5/src/Task2/Structure/src/stct/Person.java
package stct;

public abstract class Person {
    private Integer m_Age = 0;
    private String m_Name = new String();

    public Person(final Integer c_Age, final String c_Name) {
        this.m_Age = c_Age;
        this.m_Name = c_Name;
    }

    public final void f_set_age(final Integer c_Age) {
        this.m_Age = c_Age;
    }

    public final Integer f_get_age() {
        return this.m_Age;
    }

    public final void f_set_name(final String c_Name) {
        this.m_Name = c_Name;
    }

    public final String f_get_name() {
        return this.m_Name;
    }
}
